examples/v1beta1/metric-strategy-example.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: "kubeflow.org/v1beta1"
+kind: Experiment
+metadata:
+  namespace: kubeflow
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: metric-strategy-example
+spec:
+  objective:
+    type: maximize
+    goal: 0.99
+    objectiveMetricName: Validation-accuracy
+    additionalMetricNames:
+      - Train-accuracy
+    metricStrategies:
+      Validation-accuracy: "max"
+      Train-accuracy: "latest"
```

Comment:
Can we avoid `-` in name?

It is not popular in Kubernetes community. Maybe we can use:

```
  - name: Train-accuracy
    value: latest
  - name: Validation-accuracy
    value: max
```
```

Reply:
@gaocegege Okay, I've applied this suggestion.